src/views/v2/day/event/venue.php

```diff
@@ -9,9 +9,10 @@
  *
  * @link http://evnt.is/1aiy
  *
- * @version 4.9.11
+ * @version TBD
```

Comment:
`@since` with the introduced version?
and a TBD with the changes introduced in TBD 😉